# HG changeset patch
# User Jean-Yves Avenard <jyavenard@mozilla.com>

Bug 1233429: Disable GMP PDM. r=cpearce

Prevent users from shooting themselves in the foot by turning some prefs on.

diff --git a/dom/media/platforms/PlatformDecoderModule.cpp b/dom/media/platforms/PlatformDecoderModule.cpp
index 530c7b9..50933ca 100644
--- a/dom/media/platforms/PlatformDecoderModule.cpp
+++ b/dom/media/platforms/PlatformDecoderModule.cpp
@@ -150,20 +150,22 @@ PlatformDecoderModule::Create()
   }
   return CreateAgnosticDecoderModule();
 }
 
 /* static */
 already_AddRefed<PlatformDecoderModule>
 PlatformDecoderModule::CreatePDM()
 {
+#ifndef RELEASE_BUILD
   if (sGMPDecoderEnabled) {
     nsRefPtr<PlatformDecoderModule> m(new GMPDecoderModule());
     return m.forget();
   }
+#endif
 #ifdef MOZ_WIDGET_ANDROID
   if(sAndroidMCDecoderPreferred && sAndroidMCDecoderEnabled){
     nsRefPtr<PlatformDecoderModule> m(new AndroidDecoderModule());
     return m.forget();
   }
 #endif
   if (sUseBlankDecoder) {
     return CreateBlankDecoderModule();
